src/components/atoms/Input/index.tsx

```diff
@@ -42,9 +53,10 @@ const Input: React.FC<InputProps> = ({
       )}
       <input
         className={cx(
-          'w-full rounded-lg bg-dark-100 text-gray-100 focus:outline-none h-[50px] pb-[4px] placeholder:text-gray-500',
+          'align-middle h-[46px] w-full rounded-lg bg-dark-100 text-gray-100 focus:outline-none placeholder:text-gray-500',
```

Comment:
add a new styles to change default style by google for autofill values. 

We can make this in tailwind 3+ using "autofill" pseudo-element.

look this exemple:

`"autofill:bg-transparent"`

you can check more about this in: [Tailwind Autofill Docs](https://tailwindcss.com/docs/hover-focus-and-other-states#autofill
)